public/app/features/alerting/unified/components/rules/EditRuleGroupModal.test.tsx

```diff
@@ -29,18 +32,27 @@ const ui = {
 };
 
 const noop = () => jest.fn();
+setupMswServer();
 
 describe('EditGroupModal', () => {
-  it('Should disable all inputs but interval when intervalEditOnly is set', async () => {
-    const namespace = mockCombinedRuleNamespace({
-      name: 'my-alerts',
-      rulesSource: mockDataSource(),
-      groups: [{ name: 'default-group', interval: '90s', rules: [], totals: {} }],
-    });
-
-    const group = namespace.groups[0];
+  const { dataSource, rulerConfig } = mimirDataSource();
+  setupDataSources(dataSource);
 
-    render(<EditRuleGroupModal namespace={namespace} group={group} intervalEditOnly onClose={noop} />);
+  it('Should disable all inputs but interval when intervalEditOnly is set', async () => {
+    const rulerGroupIdentifier: RuleGroupIdentifier = {
+      dataSourceName: dataSource.name,
+      groupName: 'default-group',
+      namespaceName: 'my-namespace',
```

Comment:
(I think Gilles is fixing these tests, but this should change to `namespace-1` to match what's in the default mimir handlers)

Reply:
Yep! I'm refactoring this test suite to use those :)

Reply:
maybe we should expose a constant to figure out which values we can use in tests? @tomratcliffe

Reply:
There is a constant for `namespace-1` already, exposed somewhere else :) we need an ongoing effort to use constants/fixtures across our tests, IMO